apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,9 +618,9 @@ var ConversationView = {
         this.draft = Threads.get(threadId).getDraft();
 
         if (this.draft) {
-          this.draft.isEdited = false;
-
           Compose.fromDraft(this.draft);
+
+          this.draft.isEdited = false;
```

Comment:
> Also ideally I'd like to see if we can just delete this.draft usage -- but not sure if it's possible. Maybe we'd still need the draft id?

Actually only ```isEdited``` prevents us from using ```draftId``` instead of ```draft```, but yeah we need ```draftId``` to at least discard draft :)

> actually, I think the "draft" mechanism should just be the model for Compose: currently, when sending, we need to extract the content from the DOM; but we should just send the current draft (granted it would itself come from the DOM because we use a contentEditable field).

Mmm, yeah, if we continuously save draft, we can easily do this since we send from ```Conversation``` (via sim selection callback) that have access to the most recent ```draft``` already.